tron/trondaemon.py

```diff
@@ -8,26 +8,18 @@
 import logging
 import logging.config
 import os
-import platform
 import signal
 
 import daemon
 import lockfile
 import pkg_resources
 import six
 from twisted.internet import defer
-from twisted.internet.main import installReactor
 from twisted.python import log as twisted_log
 
 import tron
 from tron.utils import flockfile
 
-if platform.system() == 'Linux':
```

Comment:
Is this simply not needed anymore with the new version of twisted? 
I'm pretty sure we originally had this here to support OSX